README.md

```diff
@@ -1,2 +1,10 @@
 # Google-Imitation
```

Comment:
Not a bad readme, but make it more informative next time to give the reader good details before they start using the application.